CMakeLists.txt

```
@@ -1,25 +1,28 @@
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.11)
 project(python_h264decoder)
 
-set(Python_ADDITIONAL_VERSIONS 2.7)
-find_package(PythonInterp 2.7 REQUIRED)
-find_package(PythonLibs 2.7 REQUIRED )
-find_package(Boost REQUIRED COMPONENTS "python")
+set(CMAKE_BUILD_TYPE Debug)
 
-include_directories(${PYTHON_INCLUDE_DIRS})
-include_directories(${Boost_INCLUDE_DIRS})
-link_directories(${Boost_LIBRARY_DIRS})
+find_package(PkgConfig REQUIRED)
+pkg_check_modules(libswscale REQUIRED libswscale)
+pkg_check_modules(libavutil REQUIRED libavutil)
+pkg_check_modules(libavcodec REQUIRED libavcodec)
 
-add_compile_options ("-std=c++0x")
+find_package(pybind11)
+if(pybind11_FOUND)
+  message("Using existing pybind11 v${pybind11_VERSION}")
+else()
+  message("Fetching latest pybind11")
+  include(FetchContent)
+  FetchContent_Declare(pybind11 GIT_REPOSITORY https://github.com/pybind/pybind11)
```

For me this fetches the dev branch. 
```suggestion
  FetchContent_Declare(
    pybind11 
    GIT_REPOSITORY https://github.com/pybind/pybind11
    GIT_TAG v2.5)
```
Would be safer to fetch a release version.